src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -130,12 +132,12 @@
   // the danger of (inadvertently) training on test data. These settings are used in the ATIRE system and also in
   // Lin et al. (ECIR 2016).
 
-  @Option(name = "-k1", handler = StringArrayOptionHandler.class, usage = "BM25 k1 parameter")
-  public String[] k1 = new String[] {"0.9"};
+  @Option(name = "-bm25.k1", handler = StringArrayOptionHandler.class, usage = "BM25: k1 parameter")
+  public String[] bm25_k1 = new String[] {"0.9"};
+
+  @Option(name = "-bm25.b", handler = StringArrayOptionHandler.class, usage = "BM25: b parameter")
+  public String[] bm25_b = new String[] {"0.4"};
```

Comment:
I found some of the Java parameters are snake_case (e.g. `bm25_b`) while others are camelCase (e.g. `bm25Accurate`).

Shall we normalize them?

Reply:
I think the convention we adopted is that underscore is used for grouping parameters together, e.g., `bm25_foo`, so `foo` are all the BM25 parameters. Otherwise, everything else is camel case?

Reply:
okay, I am fine with it -- they just look weird imo.

Reply:
Oh, I found `bm25Accurate`